tools/markdown_lint/cli.py

```
@@ -0,0 +1,337 @@
+"""Command-line interface for the markdown linter."""
+
+import argparse
+import json
+import sys
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)